src/SkyApm.Diagnostics.HttpClient/Handlers/DefaultRequestDiagnosticHandler.cs

```diff
@@ -47,7 +47,7 @@ public bool OnlyMatch(HttpRequestMessage request)
 
         public void Handle(ITracingContext tracingContext, HttpRequestMessage request)
         {
-            var operationName = request.RequestUri.GetLeftPart(UriPartial.Path);
+            var operationName = request.RequestUri.GetComponents(UriComponents.Path | UriComponents.KeepDelimiter, UriFormat.UriEscaped);

```

Comment:
What does this mean?

Reply:
assuming there is an URI `https://en.wikipedia.org/wiki/github,` old code returns `https://en.wikipedia.org/wiki/github` while new code returns `/wiki/github.`